### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by luklue**
@Citoxe and me created a ReadMe for better usability! 😋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# LeProxy
+
+LeProxy is the HTTP/SOCKS proxy server for everybody!
+
+LeProxy is designed for anonymous surfing, improved security and privacy
```
Comment (Citoxe) [inline README.md]: improved security privacy, and

File: README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+and circumventing geoblocking, so that you can enjoy the web like it's
```
Comment (Citoxe) [inline README.md]: bypass geoblocking

File: README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+meant to work and access your favorite online video platform without
+annoying country blocks while traveling.
+
+LeProxy is a powerful, lightweight, fast and simple to use proxy server
+that you can host on your own server or PC at home and then access from anywhere.
+It supports optional authentication so you can share a server instance
+with your family and friends without having to worry about third parties.
+It provides maximum compatibility with a large number of clients and
```
Comment (Citoxe) [inline README.md]: high

File: README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+## Usage
+
+Once [installed](#install), you can start LeProxy by simply running:
+
+```bash
+$ php leproxy.php 
+```
+
+By default, LeProxy will listen on the address `127.0.0.1:1080`. 
+If you want to listen on another address, you can supply an explicit listen address.
```
Comment (Citoxe) [inline README.md]: enter

File: README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+For example, if you want to listen on all interfaces and allow access to LeProxy
+from the outside:
+
+```bash
+$ php leproxy.php 0.0.0.0:1080
+```
+
+Note that LeProxy does not require authentication by default,
+so the above should we used with care.
```
Comment (Citoxe) [inline README.md]: be

File: README.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+## Install
+
+The recommend way to install LeProxy is to clone (or download) this repository
+and use [Composer](http://getcomposer.org) to download its dependencies.
+Therefore you'll need PHP, git and curl installed:
+
+```bash
+$ sudo apt-get install php7.0 php7.0-cli git curl
+$ git clone https://github.com/leproxy/leproxy.git
+$ curl -s https://getcomposer.org/installer | php
```
Comment (legionth) [inline README.md]: Does this really need `curl` to be installed? I know curl ist needed for the acceptance test, but is it needed to install leproxy?

**Pull request merged**